.github/workflows/darwin.yaml

```diff
@@ -26,14 +26,25 @@ concurrency:
 
 env:
     CHIP_NO_LOG_TIMESTAMPS: true
-    
```

Comment:
Adding a comment to hold until I can update the build requirements

Reply:
The "Build Darwin" job with dependencies on the other jobs works for now, I'll take that out once you've changed the project build requirements.